app/presenters/curation_concerns/single_use_link_presenter.rb

```diff
@@ -0,0 +1,35 @@
+module CurationConcerns
+  class SingleUseLinkPresenter
+    include ActionView::Helpers::TranslationHelper
+
+    attr_reader :link, :link_type, :url_helper
+
+    delegate :downloadKey, :expired?, :to_param, to: :link
+
+    # @param link [SingleUseLink]
+    def initialize(link)
+      @link = link
+      define_link
+    end
+
+    def human_readable_expiration
+      hours = (link.expires - Time.zone.now).to_i / 3600
+      return t('curation_concerns.single_use_links.expiration.lesser_time') if hours < 1
+      t('curation_concerns.single_use_links.expiration.time', value: hours)
+    end
+
+    def short_key
+      link.downloadKey.first(6)
+    end
+
+    def define_link
+      if link.path =~ /downloads/
+        @link_type = t('curation_concerns.single_use_links.download.type')
```

Comment:
@awead Extract that to its own private method, like:

```ruby
def download_link?
  link.path =~ /downloads/
  # or something that returns true/false instead of truthy/falsey like link.path.include? 'downloads' ?
end
```

I'd rather see duplication of a conditional than more ivars.